packages/vaadin-template-renderer/src/vaadin-template-renderer.js

```diff
@@ -6,14 +6,30 @@ import { Templatizer } from './vaadin-template-renderer-templatizer.js';
 function createRenderer(template) {
   const templatizer = Templatizer.create(template);
 
-  return (root, _owner, model) => {
-    template.__templatizer = templatizer;
+  const renderer = (root, _owner, model) => {
     template.__templatizer.render(root, model);
   };
+
+  template.__templatizer = templatizer;
+  renderer.__templatized = true;
+
+  return renderer;
+}
+
+function assignRenderer(component, rendererName, template) {
+  const oldRenderer = component[rendererName];
+
+  if (oldRenderer && !oldRenderer.__templatized) {
+    const tag = component.tagName.toLowerCase();
```

Comment:
nit: can skip the `toLowerCase()` if you use `localName`
```suggestion
    const tag = component.localName;
```

Reply:
What do you think about the error message? 

I'm figuring out how to mention the renderer name there, but haven't come up with a suitable message so far.

Reply:
This looks informative enough imo